src/platform_impl/windows/event_loop.rs

```diff
@@ -1218,7 +1218,12 @@ unsafe fn public_window_callback_inner<T: 'static>(
                     });
                 }
             }
-            0
+
+            if msg == WM_SYSCHAR {
```

Comment:
Good point. I've also added a handler for `WM_MENUCHAR` to disable the annoying beep sound when an ALT+keypress is entered. This could in principle be used to communicate an event to be used in a menu system, but not sure how generalisable that would be to other platforms.